projpicker/web/index.html

```diff
@@ -0,0 +1,288 @@
+                       name="logical_op" onclick="onClick(this)" checked>
+                <label for="and">AND</label>
+            </fieldset>
+            <fieldset>
+                <input type="radio" id="or" value="or"
+                       name="logical_op" onclick="onClick(this)">
+                <label for="or">OR</label>
+            </fieldset>
+            <fieldset>
+                <input type="radio" id="xor" value="xor"
+                       name="logical_op" onclick="onClick(this)">
+                <label for="xor">XOR</label>
+            </fieldset>
+        </form>
+    </div>
+</body>
+<script>
+    var map = L.map(
+        'map', {
+            center: [34.2347566, -83.8676613],
+            crs: L.CRS.EPSG3857,
+            zoom: 5,
+            zoomControl: true,
+            preferCanvas: false,
+        }
+    );
+
+    var tileLayer = L.tileLayer(
+        'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+            'attribution': 'Data by \u0026copy; \u003ca href="http://openstreetmap.org"\u003eOpenStreetMap\u003c/a\u003e, under \u003ca href="http://www.openstreetmap.org/copyright"\u003eODbL\u003c/a\u003e.',
+            'detectRetina': false,
+            'maxNativeZoom': 18,
+            'maxZoom': 18,
+            'minZoom': 0,
+            'noWrap': true,
+            'opacity': 1,
+            'subdomains': 'abc',
+            'tms': false
+        }
+    ).addTo(map);
+
+    // FeatureGroup is to store editable layers.
+    var geomsLayer = new L.geoJSON().addTo(map);
+
+    new L.Control.Draw({
+        position: 'topleft',
+        draw: {
+            'polyline': {
+                'allowIntersection': false
+            },
+            circle: false,
+            circlemarker: false,
+            polyline: false
+        },
+        edit: {
+            'poly': {
+                'allowIntersection': false
+            },
+            'featureGroup': geomsLayer
+        },
+    }).addTo(map);
+
+    map.on(L.Draw.Event.CREATED, async function (e) {
+        geomsLayer.addLayer(e.layer);
+        await sendQuery();
+
+        await getData('/projdata')
+            .then(data => populateCRSList(Object.keys(data)))
+    });
+
+    function populateCRSList(crs_keys) {
+        const list = document.getElementById('crslist');
+        while (list.firstChild) {
+            list.removeChild(list.firstChild);
+        }
+        const listItems = crs_keys.map( function(element) {
+                return `<li tabindex='1' id='${element}' onclick='onCRSSelect()'>${element}</li>` });
+
+        list.innerHTML = listItems.join('');
+    }
+
+    async function getData(url) {
+      var response = await fetch(url);
+      return response.json();
+    }
+
+    async function onCRSSelect() {
+        var selectedId = document.activeElement.id;
+        console.log(selectedId);
+        var data = await getData('/projdata');
+        console.log(data)
+        selectedCRS = data[selectedId];
```

Comment:
This is always tricky. Two-letter acronyms vs. Multi-letter acronyms. .NET would use `Id` and `CRS`, not `ID` and `CRS`, but it's all debatable. `selectedId` vs `selectedID`, `selectedCRS` vs `selectedCrs`. I think as is they look OK (`selectedId` and `selectedCRS`).